Library/Homebrew/formula_creator.rb

```diff
@@ -153,11 +153,15 @@ class #{Formulary.class_s(name)} < Formula
           def install
             # ENV.deparallelize  # if your formula fails when building in parallel
         <% if mode == :cmake %>
-            system "cmake", ".", *std_cmake_args
+            # Use `"-G", "Ninja"` to build with ninja
```

Comment:
I'm leaning towards omitting this. I would say `ninja` should only be used if it makes a significant difference. Otherwise it's really slower from having to pull in several dependencies.